File: CSharp/k01.cs
using System;

namespace CSharp
{
    public class K01 : IExecutable
    {

        public override void Question01()
        {
            Console.WriteLine("Hello World! ようこそC#言語の世界へ！");
        }

        public override void Question02()
        {
            Console.WriteLine("Hello World!");
            Console.WriteLine("ようこそ");
            Console.WriteLine("C#言語の世界へ！");
        }

        public override void Question03()
        {
            Console.WriteLine("整数：" + 12345);
            Console.WriteLine("実数：" + 123.456789);
            Console.WriteLine("文字：" + 'X');
            Console.WriteLine("文字列：" + "ABCdef");
        }

        public override void Question04()
        {
            Console.WriteLine("              ##");
            Console.WriteLine("             #  #");
            Console.WriteLine("             #  #");
            Console.WriteLine("            #    #");
            Console.WriteLine("           #      #");
            Console.WriteLine("         ##        ##");
            Console.WriteLine("       ##            ##");
            Console.WriteLine("    ###                ###");
            Console.WriteLine(" ###       ##    ##       ###");
            Console.WriteLine("##        #  #  #  #        ##");
            Console.WriteLine("##         ##    ##         ##");
            Console.WriteLine(" ##     #            #     ##");
            Console.WriteLine("  ###     ##########     ###");
            Console.WriteLine("     ###              ###");
            Console.WriteLine("        ##############");
        }
    }
}

File: Kotlin/src/K01.kt
class K01 : IExectuable() {

    override fun question_01() {
        println("Hello World! ようこそKotlin言語の世界へ！")
    }

    override fun question_02() {
        println("Hello World!")
        println("ようこそ")
        println("Kotlin言語の世界へ！")
    }

    override fun question_03() {
        println(String.format("整数：%d", 12345))
        println(String.format("実数：%f", 123.456789))
        println(String.format("文字：%c", 'A'))
        println(String.format("文字列：%s", "ABCdef"))
    }

    override fun question_04() {
        println("              ##")
        println("             #  #")
        println("             #  #")
        println("            #    #")
        println("           #      #")
        println("         ##        ##")
        println("       ##            ##")
        println("    ###                ###")
        println(" ###       ##    ##       ###")
        println("##        #  #  #  #        ##")
        println("##         ##    ##         ##")
        println(" ##     #            #     ##")
        println("  ###     ##########     ###")
        println("     ###              ###")
        println("        ##############")
    }
}